### PR TITLE
fix: return success if ns,quota,pvc exist while open worker cluster

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:
- return success if ns,quota,pvc exist while open worker cluster
- return a specific error to indicate creating control cluster integration error while creating tenant
- do not open the control cluster to run workflow while creating tenant
- some refactor about give default value to cyclone server config

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @cd1989 @supereagle 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/server/config/config.go [hunk 1]
```diff
@@ -65,12 +67,19 @@ func LoadConfig(cm *core_v1.ConfigMap) error {
...
 
+	modifier(&Config)
+
 	log.Infof("cyclone server config: %v", Config)
 	return nil
 }
 
 // validate validates some required configurations.
 func validate(config *CycloneServerConfig) bool {
+	return true
```
Comment (supereagle) [inline pkg/server/config/config.go]: Always return true?

File: pkg/server/config/config.go [hunk 2]
```diff
@@ -81,5 +90,19 @@ func validate(config *CycloneServerConfig) bool {
...
 	}
 
-	return true
+	if config.DefaultPVCConfig.Size == "" {
+		log.Warningf("DefaultPVCConfig.Size not configured, will use default value '5Gi'")
+		config.DefaultPVCConfig.Size = common.DefaultPVCSize
+	}
+
+	if config.WorkerNamespaceQuota == nil {
+		log.Warningf("WorkerNamespaceQuota not configured, will use default value limits.cpu limits.memory requests.cpu requests.memory")
```
Comment (supereagle) [inline pkg/server/config/config.go]: What about `WorkerNamespaceQuota not configured, will use default quota`?

Comment (supereagle) [reply pkg/server/config/config.go]: Use `Warningln` instead of `Warningf`, as there is no format.

Comment (supereagle) [reply pkg/server/config/config.go]: Use `Warningln` instead of `Warningf`, as there is no format.

File: pkg/server/config/config.go (hunk 1 above)
Comment (zhujian7) [reply pkg/server/config/config.go]: There is nothing we need to validate by now, I think I need to delete this function at this time.

File: pkg/server/config/config.go (hunk 2 above)
Comment (zhujian7) [reply pkg/server/config/config.go]: Ok, i will use `Warningln`

Comment (zhujian7) [reply pkg/server/config/config.go]: What do you mean `default quota`? empty?

Comment (zhujian7) [reply pkg/server/config/config.go]: Ok, i will use `Warningln`

Comment (supereagle) [reply pkg/server/config/config.go]: > What do you mean `default quota`? empty?

Just change the log content to make it simpler.

File: pkg/server/config/config.go (hunk 1 above)
Comment (supereagle) [reply pkg/server/config/config.go]: ok

**Comment by zhujian7** [comment]
@supereagle PTAL

**Pull request merged**